frontends/election-manager/src/app.test.tsx

```diff
@@ -359,6 +369,25 @@ test('L&A (logic and accuracy) flow', async () => {
       expect.stringContaining(LogEventId.TestDeckPrinted)
     )
   );
+  expect(mockKiosk.log).toHaveBeenCalledWith(
+    expect.stringContaining('BMD paper ballot test deck')
+  );
+  expect(container).toMatchSnapshot();
+  jest.advanceTimersByTime(30000);
+
+  // L&A package: HMPB test deck
+  await screen.findByText('Printing L&A Package: District 5', {
+    exact: false,
+  });
+  expect(printer.print).toHaveBeenCalledTimes(4);
+  await waitFor(() =>
+    expect(mockKiosk.log).toHaveBeenCalledWith(
+      expect.stringContaining(LogEventId.TestDeckPrinted)
+    )
+  );
+  expect(mockKiosk.log).toHaveBeenCalledWith(
+    expect.stringContaining('Hand-marked paper ballot test deck')
+  );
```

Comment:
The snapshots throughout this test provide solid coverage, but I do plan to return to these tests to add explicit assertions to make what we're testing clearer, e.g. assertions for expected tally counts, etc.

https://github.com/votingworks/vxsuite/issues/1867